release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,13 @@
+## Version 1.2.0.0 2021-11-04
+
+Compatible with OpenSearch Dashboards 1.2.0
+
+### Maintenance
+* Bumps version to 1.2 ([#128](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/128))
+* Cherry-pick commits from main branch to 1.x branch ([#131](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/131))
```

Comment:
This was not new changes for 1.2. Just a cherry pick from main to 1.x for backporting

Reply:
Will remove this, thank you!